geopandas install issue?

revert #18786 

https://travis-ci.org/pandas-dev/pandas/jobs/316397406

```
./../../miniconda3/envs/pandas/lib/python3.6/site-packages/geopandas/__init__.py:4: in <module>
    from geopandas.io.file import read_file
../../../miniconda3/envs/pandas/lib/python3.6/site-packages/geopandas/io/file.py:3: in <module>
    import fiona
../../../miniconda3/envs/pandas/lib/python3.6/site-packages/fiona/__init__.py:69: in <module>
    from fiona.collection import Collection, BytesCollection, vsi_path
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    import logging
    import os
    import warnings
    
    from fiona import compat
>   from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
E   ImportError: libpoppler.so.71: cannot open shared object file: No such file or directory
```

cc @jorisvandenbossche

--- a/pandas/tests/test_downstream.py
+++ b/pandas/tests/test_downstream.py
@@ -92,7 +92,6 @@ def test_pandas_datareader():
     pandas_datareader.get_data_google('AAPL')
 
 
-@pytest.mark.xfail(reason="install not working, gh-18780")
 def test_geopandas():
 
     geopandas = import_module('geopandas')  # noqa

